src/main/java/emu/grasscutter/command/commands/SwitchElementCommand.java

```diff
@@ -0,0 +1,63 @@
+package emu.grasscutter.command.commands;
+
+import emu.grasscutter.GameConstants;
+import emu.grasscutter.Grasscutter;
+import emu.grasscutter.command.Command;
+import emu.grasscutter.command.CommandHandler;
+import emu.grasscutter.data.GameData;
+import emu.grasscutter.data.excels.AvatarData;
+import emu.grasscutter.game.avatar.Avatar;
+import emu.grasscutter.game.player.Player;
+
+import java.util.List;
+
+@Command(label = "switchelement", usage = "switchelement [White/Anemo/Geo/Electro]", aliases = {"se"}, permission = "player.switchElement", threading = true, description = "commands.switchElement.description")
+public class SwitchElementCommand implements CommandHandler {
+    @Override
+    public void execute(Player sender, Player targetPlayer, List<String> args) {
+        if (args.size() != 1) {
+            CommandHandler.sendTranslatedMessage(sender, "commands.switchElement.usage");
+            return;
+        }
+        if (sender == null) {
+            Grasscutter.getLogger().info("SwitchElement command couldn't be called by console.");
+            return;
+        }
+
+        String element = args.get(0);
+
+        Integer elementId = switch (element) {
```

Comment:
This would preferrably be case insensitive.

Reply:
1